test('Hello',()=>{

})

test('Fail',()=>{
    throw new Error('Failure')
})